plugins/source/heroku/main.go

```diff
@@ -6,7 +6,5 @@ import (
 )
 
 func main() {
-	serve.Serve(serve.Options{
-		SourcePlugin: plugin.Plugin(),
-	})
+	serve.Source(plugin.Plugin())
```

Comment:
Should we put sentry here as well?

Reply:
eventually yes, but there's an issue with sentry login

Reply:
Going to merge this PR to unblock and we can add the sentry DSNs later